libs/sdk-ui-dashboard/src/converters/filterConverters.ts

```diff
@@ -65,9 +67,9 @@ export function filterContextAttributeFilterToAttributeFilter(
  *
  * @param filter - filter context attribute filter to convert
  * @param widget - widget to use to get dateDataSet for date filters
- * @internal
+ * @public
  */
-export function filterContextDateFilterToDateFilter(
+export function dashboardDateFilterToDateFilter(
     filter: IDashboardDateFilter,
     widget: IWidgetDefinition,
```

Comment:
Maybe we could also use a version that does not take widget, but a date attribute? Because the user "outside" of the component might want to use this not for some widget.

Reply:
Created mew functions taking dateDataSet ObjRef object instead of the widget. These are distinguished by name, either you can convert byWidget or byDateDataSet.